components/Header/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Link from 'next/link';
+import styled from 'styled-components';
+
+const SPAN = styled.span`
```

Comment:
Aqui não precisa ser tudo maiúsclão, você pode escrever como se estivesse criando um component mesmo, o que no final, não deixa de ser.
```suggestion
const Span = styled.span`
```

Reply:
done